src/Microsoft.Health.Dicom.SqlServer/Features/ExtendedQueryTag/ExtendedQueryTagDataRowsBuilderV2.cs

```diff
@@ -0,0 +1,123 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using Dicom;
+using EnsureThat;
+using Microsoft.Health.Dicom.Core.Extensions;
+using Microsoft.Health.Dicom.Core.Features.ExtendedQueryTag;
+using Microsoft.Health.Dicom.SqlServer.Features.Schema.Model;
+
+namespace Microsoft.Health.Dicom.SqlServer.Features.ExtendedQueryTag
+{
+    /// <summary>
+    /// Class that build ExtendedQueryTagRows.
+    /// </summary>
+    internal static class ExtendedQueryTagDataRowsBuilderV2
```

Comment:
While we are following the pattern of versioning the various stores to maintain backward compatibility, I don't think we need to version the builder and the data transfer object. We should be able to simply add the static methods to the builder and properties to the rows DTO.

Reply:
Fixed this. I have used a new schema version constant and used that in the builder for considering the correct field.